api/app/src/main/kotlin/packit/model/dto/RunInfoDto.kt

```diff
@@ -3,7 +3,7 @@ package packit.model.dto
 data class RunInfoDto(
     val taskId: String,
     val packetGroupName: String,
-    val status: String,
+    val status: String? = null,
```

Comment:
The `RunInfoDto` is just the `BasicRunInfoDto` with logs? And we don't use inheritance because data classes don't get on with inheritance?

I guess status could arguably be an enum.

Reply:
agreed, ill make status an enum and i think both you and anmol (in prev PR) pointed out that the basicRunInfo and runInfo are a bit weird, ive just removed the basic one and the logs field can just be null in the `RunInfoDto`

Reply:
oh btw enums are from https://github.com/mrc-ide/rrq/blob/ba2784057e8ad83e304ac9ae44081110c6b3d19c/R/common.R